toDoList.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>To-Do List</title>
+        <link rel='stylesheet' href='https://use.fontawesome.com/releases/v5.0.6/css/all.css'>
+    </head>
+    <body>
+
+        <div id='header'>
+            <img class='logo' id='logo' src='EventMobiLogo.svg' alt='EventMobi'>
+        </div>
+
+        <div id='content'>
+
+            <div>
+                Event Planner To-Do List
+            </div>
+
+            <div class='to-do-item'>
```

Comment:
Why are we committing a static todo to the repo? The todo list should start out empty should it not?